docs/pages/database-access/guides/rds.mdx

```diff
@@ -135,12 +137,60 @@ Teleport database agent will be using (user or role).
   }
   ```
   </TabItem>
+  <TabItem label="RDS proxy / IAM user">
+  Use this policy if you're connecting to RDS proxies and your Teleport
+  database agent runs as an IAM user (for example, it uses an AWS credentials
+  file).
+  ```json
```

Comment:
Please take into account @gabrielcorado's docs updates in https://github.com/gravitational/teleport/pull/10451.